src/wp-includes/interactivity-api/interactivity-api.php

```diff
@@ -66,7 +66,11 @@ function wp_interactivity_process_directives_of_interactive_blocks( array $parse
 
 	return $parsed_block;
 }
-add_filter( 'render_block_data', 'wp_interactivity_process_directives_of_interactive_blocks' );
+/*
```

Comment:
There is code to update above to align the approach:

```PHP
			/*
			 * Uses a priority of 100 to ensure that other filters can add additional
			 * directives before the processing starts.
			 */
			add_filter( 'render_block_' . $block_name, $process_interactive_blocks, 100, 2 );

Reply:
@gziolo Not sure I follow, are you suggesting making a code change elsewhere? Feel free to make the edit directly in that case.

Reply:
@c4rl0sbr4v0 addressed it. All good on my side 💯